pkg/testhelper/kube/kube.go

```diff
@@ -73,14 +75,18 @@ func NewKubeClient(config *Config) *Kube {
 	scheme := runtime.NewScheme()
 	utilruntime.Must(corev1.AddToScheme(scheme))
 	utilruntime.Must(appsv1.AddToScheme(scheme))
-	utilruntime.Must(apiv1.AddToScheme(scheme)) // add OnePasswordItem to scheme
```

Comment:
How do we make sure it behaves the same in terms of adding the appropriate group version (from [`api/v1/groupversion_info.go`](https://github.com/1Password/onepassword-operator/blob/main/api/v1/groupversion_info.go))? Changing it to `admissionregistrationv1` feels a bit off, even though it might be working.